### DISCUSSION TITLE
Can the logo be clickable to return to the home page?

### DISCUSSION THREAD

**Discussion by seahagcafe** [jpanther/congo#12] [Q&A] [answered]
I want to implement the logo option that renders in the top left such that it's clickable and will return users to the home page. I'm not seeing an href in the header.html partial, but I'm happy to add one if others would find it useful.

Thanks!

**Comment by jpanther** [comment, marked as answer]
Oh wow, this is definitely an oversight! I'll fix this in the next release.

**Reply by jpanther** [reply]
This is now fixed in v1.4.0
